orderfilter/filter_test.go

```diff
@@ -21,9 +21,31 @@ var (
 	contractAddresses                  = ethereum.GanacheAddresses
 )
 
+func generateDecodedFilter(chainID int, customOrderSchema string, contractAddresses ethereum.ContractAddresses) (*Filter, error) {
```

Comment:
Can you add a comment explaining the approach used here and how `generateDecodedFilter` fits in?